packages/medusa/src/loaders/defaults.ts

```diff
@@ -1,4 +1,8 @@
-import { BasePaymentService, BaseNotificationService, BaseFulfillmentService } from 'medusa-interfaces'
```

Comment:
issue: Don't think it is part of the scope of that pr

Reply:
Yes but you shouldn’t lint the entire core for example as it is not part of that scoped ☺️

Reply:
done reverted